src/testing/flow_solver_cases/naca0012.cpp

```diff
@@ -71,7 +71,7 @@ void NACA0012<dim,nstate>::set_higher_order_grid(std::shared_ptr<DGBase<dim, dou
     for (unsigned int i=0; i<this->all_param.grid_refinement_study_param.num_refinements; ++i) {
         dg->high_order_grid->refine_global();
     }
```

Comment:
```suggestion
    }
    dg->allocate_system();
```

Reply:
Not specific to this PR but since we are fixing other issues in FlowSolver it may as well go here: I have noticed that the wrong number of degrees of freedom is printed unless dg->allocate_system() is called after refining the grid and before calling dof_handler.ndofs(). This probably applies to all other cases as well.

Reply:
fixed